Python/bytecodes.c

```diff
@@ -5245,21 +5268,44 @@ dummy_func(
         tier2 op(_EXIT_TRACE, (exit_p/4 --)) {
             _PyExitData *exit = (_PyExitData *)exit_p;
         #if defined(Py_DEBUG) && !defined(_Py_JIT)
-            _Py_CODEUNIT *target = _PyFrame_GetBytecode(frame) + exit->target;
+            const _Py_CODEUNIT *target = ((frame->owner >= FRAME_OWNED_BY_INTERPRETER)
```

Comment:
I'd actually prefer to drop this whole block. It is logging the wrong thing (execution) at the wrong level (2) and making assumptions about the state of VM rather than just logging what it does.

If we are logging at level 4, we'll get the `_EXIT_TRACE` in the log.